nss-evening-cohort-13/Spinnovations
# Payment crud

## Description
add, update, and edit payment info. 

To access, go to profile, then my payment info


## How Can This Be Tested?
git fetch origin payment-crud

**be sure to make database adjustments as I added in an "active" column in the paymentInfo table

the file should be available in our slack channel. 

## Types of changes
